lib/src/r_tree/leaf_node.dart

```diff
@@ -57,7 +61,7 @@ class LeafNode<E> extends Node<E> {
   }
 
   clearChildren() {
```

Comment:
Since this will be a major, I'd really like to clean up all methods and explicitly declare their return types.

Reply:
![image](https://github.com/Workiva/r_tree/assets/33633765/f071720a-71eb-41ea-9433-fe9f5a5e5c38)